docs/source/user_guide.rst

```diff
@@ -0,0 +1,50 @@
+User Guide
+==========
+Resfo is an abbreviation for "Results Fortran Output"
+
+Purpose
+-------
+OPM FLOW outputs files on various formats. Resfo is parsing these files and read them
+into Python objects. Resfo-utilities is a toolkit to work with the output from Resfo.
+
+OPM FLOW output format
+----------------------
+The output files of OPM flow follows a pattern where the file is divided into entries.
```

Comment:
This section about format could just link to https://resfo.readthedocs.io/en/latest/the_file_format.html